client/locales/ko.json

```diff
@@ -0,0 +1,7892 @@
+  },
+  "course.assessment.submission.logs.userSessionId": {
+    "defaultMessage": "사용자 세션 토큰"
+  },
+  "course.assessment.submission.mark": {
+    "defaultMessage": "게시를 위한 제출"
+  },
+  "course.assessment.submission.maximumGroupGrade": {
+    "defaultMessage": "이 그룹의 최대 등급"
+  },
+  "course.assessment.submission.multiplier": {
+    "defaultMessage": "배수"
+  },
+  "course.assessment.submission.noAnswerSelected": {
+    "defaultMessage": "선택된 이전 답변이 없습니다."
+  },
+  "course.assessment.submission.ok": {
+    "defaultMessage": "확인"
+  },
+  "course.assessment.submission.pastAnswers": {
+    "defaultMessage": "이전 답변"
+  },
+  "course.assessment.submission.point": {
+    "defaultMessage": "포인트"
+  },
+  "course.assessment.submission.pointGrade": {
+    "defaultMessage": "이 포인트의 등급"
+  },
+  "course.assessment.submission.privateTestCaseFailure": {
+    "defaultMessage": "귀하의 코드는 하나 이상의 비공개 테스트 케이스를 통과하지 못했습니다."
+  },
+  "course.assessment.submission.publicTestCaseFailure": {
+    "defaultMessage": "귀하의 코드는 하나 이상의 공개 테스트 케이스를 통과하지 못했습니다."
+  },
+  "course.assessment.submission.publish": {
+    "defaultMessage": "등급 공개"
+  },
+  "course.assessment.submission.publishConfirmation": {
+    "defaultMessage": "모든 {graded} 채점된 제출물을 공개하시겠습니까 ({selectedUsers})? 이 조치는 되돌릴 수 없습니다! 모든 채점된 제출물이 공개되며 사용자는 자신의 등급을 볼 수 있습니다."
+  },
+  "course.assessment.submission.publishJobPending": {
+    "defaultMessage": "제출물이 현재 공개 중입니다. 기다려 주세요."
+  },
+  "course.assessment.submission.publishSuccess": {
+    "defaultMessage": "위의 모든 채점된 제출물이 공개되었습니다."
+  },
+  "course.assessment.submission.published": {
+    "defaultMessage": "채점됨"
+  },
+  "course.assessment.submission.question": {
+    "defaultMessage": "질문"
+  },
+  "course.assessment.submission.questionNumber": {
+    "defaultMessage": "Q{number}"
+  },
+  "course.assessment.submission.readOnlyEditor.expandComments": {
+    "defaultMessage": "모든 댓글 확장"
+  },
+  "course.assessment.submission.readOnlyEditor.showCommentsPanel": {
+    "defaultMessage": "댓글 패널 보기"
+  },
+  "course.assessment.submission.reevaluate": {
+    "defaultMessage": "답변 재평가"
+  },
+  "course.assessment.submission.rendererNotImplemented": {
+    "defaultMessage": "이 질문 유형에 대한 디스플레이가 아직 구현되지 않았습니다."
+  },
+  "course.assessment.submission.requestFailure": {
+    "defaultMessage": "요청 처리 중 오류가 발생했습니다."
+  },
+  "course.assessment.submission.reset": {
+    "defaultMessage": "답변 재설정"
+  },
+  "course.assessment.submission.resetConfirmation": {
+    "defaultMessage": "답변을 재설정하시겠습니까? 이 조치는 되돌릴 수 없으며 이 질문에 대한 모든 현재 작업이 손실됩니다."
+  },
+  "course.assessment.submission.runCode": {
+    "defaultMessage": "코드 실행"
+  },
+  "course.assessment.submission.runCodeWithLimit": {
+    "defaultMessage": "코드 실행 ({attemptsLeft, plural, one {# 시도} other {# 시도}} 남음)"
+  },
+  "course.assessment.submission.saveDraft": {
+    "defaultMessage": "초안 저장"
+  },
+  "course.assessment.submission.saveGrade": {
+    "defaultMessage": "등급 저장"
+  },
+  "course.assessment.submission.savedAt": {
```

Comment:
#9659 removed some translations.
Ensure these are removed in yours too.
https://github.com/Coursemology/coursemology2/pull/7659/files#diff-2e6669fb7d6cf2ec9e23dd3fda6a1f8bf7efd4be5764c04a99c47d72f5f0c118